connectors/src/api/webhooks/webhook_slack.ts

```diff
@@ -427,6 +512,19 @@ const _webhookSlackAPIHandler = async (
   }
 };
 
+function isChannelNameWhitelisted(
```

Comment:
What about moving this to a dedicated file so the logic for auto add channel lives in one unique place?